examples/gpio/gpio_nuvoton.c

```diff
@@ -134,7 +140,76 @@ int TPM2_GPIO_Nuvoton_Example(void* userCtx, int argc, char *argv[])
     /* GPIO un-configuration is done using NVDelete, no further action needed */
     /* Nuvoton can reconfigure any GPIO without deleting the created NV index */
     if (gpioMode == NUVOTON_GPIO_MODE_UNCONFIG) {
-        printf("Reconfiguration does not require to NV index deletion\n");
+        printf("Deleting GPIO NV Index\n");
+
+        XMEMSET(&tpmSessionIndex, 0, sizeof(tpmSessionIndex));
```

Comment:
The memset calls needs to be at top, so `wolfTPM2_UnloadHandle` do not use garbage handles.